### ISSUE TITLE
[docu] gifs that explain deconvolution intuitively

### ISSUE THREAD

**Issue by behinger** [unfoldtoolbox/unfold#72]
I'm thinking of a series of gifs that explain convolution / deconvolution

1. Event "spikes" convolved with a simple ERP-like pattern => + add continuous noise

1. Estimate two ERPs and change overlap (show % overlap for each)

1. Unfold Estimate in Relation to number of events, compare to ERP (should be more or less identical except shape differences)

**Comment by behinger** [comment]
first try for 1:
![grafik](https://user-images.githubusercontent.com/10183650/61333451-3fdab700-a827-11e9-95bf-4e31924b131f.png)

**Comment by olafdimigen** [comment]
Animation: Good idea. But I only see a blank .PNG graphic without content above.

**Comment by behinger** [comment]
ha! true ok next try:
![grafik](https://benediktehinger.de/upload/unfold_explanation.gif)

**Comment by behinger** [comment]
New colors & slower
![grafik](https://benediktehinger.de/upload/unfold_explanation_v2.gif)

**Comment by behinger** [comment]
Next graphic is there :-) Not really as I exactly intended, but this one shows the difference between deconv & no deconv (classic)
![grafik](https://benediktehinger.de/upload/unfold_explanation2.gif)

**Comment by olafdimigen** [comment]
nice. some minor suggestions:

animation 1: 
- minor layout: remove spatial overlap between FIR sticks and true response (just make sticks shorter?)
animation 2: 
- rename "classic ERP" to "simple averaging" or just "averaging"?
- the two stick function time ranges on top are not intuitive. why are the latencies of the "early" events/stick functions changing as the number of observations increases?
- classic ERP needs a bit more y-axis, I am getting a mild case of axis cut-off anxiety
- show true responses to be able to see what differences is expected between clue and orange?

**Comment by behinger** [comment]
yeah the function ranges, I'm still not sure what to do.

The first one is over the whole experiment "70s max", the second one shows that the distance between events is increased (because the number of events is fixed at 50.)
I want to add a "zoom-in trichter" then the relationship should become clear. But not sure when I will manag eto do so :D

True response would add yet another panel, I feel I always have too many panels already :-D

Also unfold erp is identical to the true one. Will fix the other things after I finished the nonlinear explanation figure

**Comment by behinger** [comment]
Non-linear graph. There is still a mistake in the second column, because the true effect is not visible.
Overall I find the figure a bit busy,and obviously it is still too fast.

I will drop many of the ticklabels, maybe change the scale between first, and second+third column for the difference
I'm also not sure why the mean-response for 2nd + 3rd is so different in magnitude

But okay, my daughter is awake :P no more giffing for now. Thanks for your feedback!

![grafik](https://benediktehinger.de/upload/unfold_explanation3.gif)

**Comment by behinger** [comment]
![grafik](https://benediktehinger.de/upload/unfold_explanation_v3.gif)
![grafik](https://benediktehinger.de/upload/unfold_explanation2_v3.gif)
![grafik](https://benediktehinger.de/upload/unfold_explanation3_v2.gif)
